packages/yourloops/lib/notifications/models.ts

```diff
@@ -0,0 +1,45 @@
+import { Session } from "../../lib/auth/models";
+import { Profile, UserRoles } from "../../models/shoreline";
+
+export enum NotificationType {
+  directshare = "careteam_invitation",
+  careteam = "medicalteam_invitation",
+  careteamPatient = "medicalteam_patient_invitation",
+}
+
+/** Some comments heres on the fields will be welcomed :-) */
+export interface INotification {

```

Comment:
This interface probably need to better match hydrophone returns values

Reply:
to be reviewed with @franck-fourel  and @yvon-dblg  as this was build from there feedback